### PR TITLE
DEVOPS-505 feat: limit replicas to one on review

### PR DESCRIPTION
**Pull request by tplessas**
Tested and all.

### CODE REVIEW

File: charts/delphai-deployment/templates/deployment.yaml [hunk 1]
```diff
@@ -14,7 +14,11 @@ metadata:
 
 spec:
   {{ if not $.Values.autoscaling.enabled }}
-  replicas: {{ $.Values.replicas }}
+  {{ if eq (lookup "v1" "ConfigMap" "default" "cluster-values").data.environment "review" }}
```
Comment (anton-delphai) [inline charts/delphai-deployment/templates/deployment.yaml:17]: Simpler
```suggestion
  {{ if eq $.clusterValues.environment "review" }}
```

Comment (tplessas) [reply charts/delphai-deployment/templates/deployment.yaml:17]: oops

File: charts/delphai-deployment/templates/deployment.yaml [hunk 2]
```diff
@@ -14,7 +14,11 @@ metadata:
 
 spec:
   {{ if not $.Values.autoscaling.enabled }}
-  replicas: {{ $.Values.replicas }}
+  {{ if eq $.clusterValues.environment "review" }}
+  replicas: 1
+  {{ else }}
+  replicas : {{ $.Values.replicas }}
+  {{ end }} {{/* if eq (lookup "v1" "ConfigMap" "default" "cluster-values").data.environment "review" */}}
```
Comment (anton-delphai) [inline charts/delphai-deployment/templates/deployment.yaml:21]:
```suggestion
  {{ end }} {{/* if eq $.clusterValues.environment "review" */}}
```

Comment (tplessas) [reply charts/delphai-deployment/templates/deployment.yaml:21]: double oops

**Review by anton-delphai — APPROVED** [review]

**Pull request merged**